app/controllers/review_controller.rb

```diff
@@ -0,0 +1,16 @@
+class ReviewController < ApplicationController
+
+  def new
+    @review=Review.new
+  end
+
+
+  def create
+    @review=Review.new(review_params)
+  end
```

Comment:
it doesnt look like you are using this either. so remove it.
Also, even if you were, you should have proper error handling per week 6. Additionally this does not persist the review at all.